week-2/mandatory/2-ecommerce-db/task.md

```diff
@@ -8,8 +8,32 @@ Below you will find a set of tasks for you to complete to set up a database for
 
 To submit this homework write the correct commands for each question here:
 ```sql
-
-
+--1
+select name,address from customers where country ='United States';
+--2
+select * from customers order by name;
+--3
+select * from products  where product_name like '%socks%';
+--4
+select p.* ,pa.unit_price,pa.supp_id from products p join product_availability pa on(p.id=pa.prod_id)  where pa.unit_price>100;
+--5
+select p.* from products p join product_availability pa on(p.id=pa.prod_id)  order by pa.unit_price desc limit 5;
+--6
+select p.product_name,pa.unit_price,s.supplier_name from products p join product_availability pa on(p.id=pa.prod_id)join suppliers s on (pa.supp_id=s.id) ;
+--7
+select p.product_name,s.supplier_name from products p join product_availability pa on(p.id=pa.prod_id)join suppliers s on (pa.supp_id=s.id) where s.country='United Kingdom' ;
+--8 
+ select o.id,o.order_reference,o.order_date,oi.quantity* pa.unit_price as total  from orders o  join order_items oi  on(o.id=oi.order_id) join product_availability pa on(oi.product_id=pa.prod_id) where o.customer_id=1 ;
```

Comment:
You have used one join condition - "oi.product_id=pa.prod_id" on the join between the order_items table and the product_availabilty table. The 2 tables have 2 columns in common and you need to add a second condition.